gsgmail_connector.py

```diff
@@ -910,7 +978,7 @@ def handle_action(self, param):
             in_json['user_session_token'] = session_id
             connector._set_csrf_info(csrftoken, headers['Referer'])
 
-        ret_val = connector._handle_action(json.dumps(in_json), None)
-        print(json.dumps(json.loads(ret_val), indent=4))
+        ph_status = connector._handle_action(json.dumps(in_json), None)
```

Comment:
You have mentioned that 'ph_status' is reserved for imports. Is this change intended then?

Reply:
completely not :D